src/plugin/open-badge/Event/BadgeEvents.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Claroline\OpenBadgeBundle\Event;
+
+final class BadgeEvents
+{
+    /**
+     * @Event("Claroline\OpenBadgeBundle\Event\AddBadgeEvent")
+     */
+    public const ADD_BADGE = 'event.funcitonal.add_badge';
```

Comment:
```suggestion
    public const BADGE_ADD = 'badge_add';
```
consistent with e.g. `TOOL_OPEN = 'tool_open';`(don't forget to update references)

Reply:
change done, thx :)